Issue templates (#1698)
TomographicImaging/CIL: .github/ISSUE_TEMPLATE/Enhancement_issue.md

---
name: Feature request
about: Request an improvement or a new feature.
title: ''
labels: 'Type: enhancement'
---
#### Describe the new feature

> <Describe the enhancement and how the software should behave once it is implemented.>

Reviewer: ```suggestion
<Describe the new feature and how the software should behave once it is implemented.>
```

	
#### Motivation behind the suggestion

<Provide information about the current behaviour and how the new feature fits into CIL.>
<Optional: provide an example showing the need of the feature.>

	
#### Optional: proposed solution

> <Describe your proposed changes to implement to new feature.>

Reviewer: ```suggestion
<Describe your proposed changes to implement the new feature.>
```

> #### Describe the new feature
> 
> <Describe the new feature and how the software should behave once it is implemented.>
> 	
> #### Motivation behind the suggestion
> 
> <Provide information about the current behaviour and how the new feature fits into CIL.>
> <Optional: provide an example showing the need of the feature.>
> 
> 	
> #### Optional: proposed solution
> 
> <Describe your proposed changes to implement the new feature.>
> <Provide links to relevant information.>
> 

Reviewer 2: would merge all of this into one short paragraph... or even leave it blank

```suggestion
```

Author: At the group meeting it seemed we were happy with this file @paskino Are we?
